### PR TITLE
Fix webtorrent styles

### PR DESCRIPTION
**Pull request by lkyuchukov**
Resolves https://github.com/brave/brave-browser/issues/22596

Screenshot before: 
Screenshot 2022-04-29 at 14 10 58

Screenshot after:
Screenshot 2022-04-29 at 12 29 30

`styles.css` was not applied at all so I deleted it. 

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [x] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [x] New files have MPL-2.0 license header
- [x] Adequate test coverage exists to prevent regressions
- [x] Major classes, functions and non-trivial code blocks are well-commented
- [x] Changes in component dependencies are properly reflected in `gn`
- [x] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [x] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by lkyuchukov** [comment]
@mariospr @mkarolin could you review this PR please ?

**Comment by rebron** [comment]
cc: @yrliou

File: components/brave_webtorrent/extension/components/torrentViewer.tsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import * as React from 'react'
 
 // Assets
 require('../../../../ui/webui/resources/fonts/muli.css')
-require('../styles/styles.css')
```
Comment (fallaciousreasoning) [inline components/brave_webtorrent/extension/components/torrentViewer.tsx:9]: Hey, I don't know that removing the stylesheet here is the right solution. I'm not too familiar with this area of the codebase, but it looks like it has a bunch of default styles to display images/video and style a few other things. Is it all unused?

File: components/brave_webtorrent/extension/styles/styles.ts [hunk 2]
```diff
@@ -9,3 +9,8 @@ export const StyledTorrentViewer = styled('div')<{}>`
   padding: 100px;
   font-family: Muli, sans-serif;
 `
+
+export const StyledButtonContainer = styled('div')<{}>`
```
Comment (fallaciousreasoning) [inline components/brave_webtorrent/extension/styles/styles.ts:13]: I think a more minimal change would be to add a ButtonsContainer element, with `flex-direction: row;` and put both buttons inside it:

```ts
export const ButtonsContainer = styled.div`
    display: flex;
    flex-direction: row;
`
```

Comment (fallaciousreasoning) [reply components/brave_webtorrent/extension/styles/styles.ts:13]: You can then use `gap: 10px` for spacing.

**Review by fallaciousreasoning — CHANGES_REQUESTED** [review]
Thanks for taking a look at this - looks much nicer now. Got a couple of requests/questions

File: components/brave_webtorrent/extension/components/torrentViewer.tsx (hunk 1 above)
Comment (lkyuchukov) [reply components/brave_webtorrent/extension/components/torrentViewer.tsx:9]: There is absolutely no difference to the torrents page when removing the style sheet. I think it's just not loaded properly as I initially tried to solve this issue with changes there.

File: components/brave_webtorrent/extension/styles/styles.ts (hunk 2 above)
Comment (lkyuchukov) [reply components/brave_webtorrent/extension/styles/styles.ts:13]: Sure, I'll change it

**Comment by fallaciousreasoning** [comment]
Man, you were totally right about the styles not being applied at all :fearful: I had a bit of a dig and I found that in [this commit](https://github.com/brave/brave-core/commit/fdc3905bfec9a21e5a56e15a9be57d85bf680483) we enabled module scope for CSS files by default. 

It looks like that was the root cause of this issue, as the CSS file was started being compiled to a module scope so the classes didn't match up properly anymore. You can give the CSS file a global (within the WebTorrent extension) scope by wrapping the entire CSS file in a `global` block:

```css
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */

:global {
    // all the styles from `style.css`
}
```

The WebTorrent page with the styles applies looks significantly better (in my opinion), and it fixes a bunch of other niggles, like huge loading spinner and incorrectly sized video elements:

image

Would you mind updating the patch to have the new fix? Thanks for digging in to this!

File: components/brave_webtorrent/extension/components/torrentViewer.tsx (hunk 1 above)
Comment (fallaciousreasoning) [reply components/brave_webtorrent/extension/components/torrentViewer.tsx:9]: Ah that's weird as - maybe the styles not being applied is the root cause of this?

**Review by fallaciousreasoning — CHANGES_REQUESTED** [review]

File: components/brave_webtorrent/extension/styles/styles.ts (hunk 2 above)
Comment (fallaciousreasoning) [reply components/brave_webtorrent/extension/styles/styles.ts]: Thanks! Hopefully this won't be necessary at all with the `:global { ... }` fix!

File: components/brave_webtorrent/extension/components/torrentViewer.tsx (hunk 1 above)
Comment (lkyuchukov) [reply components/brave_webtorrent/extension/components/torrentViewer.tsx:9]: That is quite likely. However, I couldn't figure out why the styles weren't applied so I made this change.

**Review by fallaciousreasoning — CHANGES_REQUESTED** [review]
LGTM but would you mind reverting the changes to `components/brave_webtorrent/extension/components/torrentViewerHeader.tsx` (it looks like it's just unrelated formatting). I'm going to trigger a build - if it passes I think we can merge.

**Comment by lkyuchukov** [comment]
Sure

**Review by fallaciousreasoning — APPROVED** [review]

**Comment by fallaciousreasoning** [comment]
Thanks for that @lkyuchukov! Sorry it took so long to get an initial review

**Pull request merged**